portal/data/courses.csv

```diff
@@ -1,4 +1,7 @@
 course_num,course_title,description,credits,teacher_id,major_id
-20,ENG 216,Technical Writing,3,53114,3
-19,MATH 126,Tech Math I,3,42267,1
-344,PHYS 106,Physics Everyday,3,42267,1
+1,ENG 216,Technical Writing,3,53114,3

```

Comment:
Do we need this extra data? If it is not needed for tests then let's get rid of it.